src/test/test-condition.c

```diff
@@ -668,6 +671,7 @@ TEST(condition_test_version) {
         ASSERT_OK_ZERO(condition_test(condition, environ));
         condition_free(condition);
 
+#if HAVE_GNU_LIBC_VERSION_H
```

Comment:
hmm, we should have a test here that verifies that checking the glibc version results in somewhat systematic behaviour if glibc is not used. probably the conditoin should always fail